apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-editor-preview.tsx

```diff
@@ -28,52 +39,48 @@ export const InAppEditorPreview = (props: InAppEditorPreviewProps) => {
         In-app template
       </div>
 
-      <Collapsible
-        open={isEditorOpen}
-        onOpenChange={setIsEditorOpen}
-        className="bg-neutral-alpha-50 border-neutral-alpha-200 flex w-full flex-col gap-2 rounded-lg border p-2"
-      >
-        <CollapsibleTrigger className="flex w-full items-center justify-between text-sm">
-          <div className="flex items-center gap-1">
-            <Code2 className="size-5" />
-            Configure preview
-          </div>
+      <InAppPreview data={previewData} isLoading={isPreviewLoading} />
```

Comment:
moved preview up like it is in the designs

Reply:
but I think it will be less jumpy if we will keep it under the payload, because the preview content height changes between the loading state and response